### PR TITLE
Approve command

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
@bot help

**Comment by davidesalerno** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by radtriste** [comment]
@bot help

**Comment by spolti** [comment]
/ bot approve

File: src/main/java/org/kiegroup/kiebot/cli/commands/BackportCommand.java [hunk 1]
```diff
@@ -14,13 +13,12 @@
...
 import io.quarkiverse.githubapp.command.airline.CliOptions;
 import io.quarkiverse.githubapp.command.airline.CommandOptions;
-import io.quarkiverse.githubapp.command.airline.Team;
 
 @Cli(name = "/backport", defaultCommand = BackportCommand.class)
 @CliOptions(defaultCommandOptions = @CommandOptions(scope = CommandOptions.CommandScope.PULL_REQUESTS), aliases = { "/cherry-pick" })
 @Command(name = "backport",
         description = "{BRANCH} or `/backport {BRANCH}` or `/cherry-pick {BRANCH}`. Add the cherry-pick/backport label so it can be backported to the given branch once the PR is merged.")
-@Team("gatekeepers")
+// @Team("gatekeepers")
```
Comment (spolti) [inline src/main/java/org/kiegroup/kiebot/cli/commands/BackportCommand.java]: uncomment?

Comment (radtriste) [reply src/main/java/org/kiegroup/kiebot/cli/commands/BackportCommand.java]: that is why I am in draft mode :D

**Comment by radtriste** [comment]
@bot help

**Review by spolti — APPROVED** [review]

**Pull request closed without merging**